packages/react/src/spinner/Spinner.tsx

```diff
@@ -25,15 +25,10 @@ export const Spinner = forwardRef<HTMLDivElement, SpinnerProps>(
   ({ children, colorScheme = "default", size = "md", ...props }, ref) => {
     return (
       <Transition duration="sm">
-        <Flex
-          alignItems="center"
-          aria-label="Loading"
-          display="inline-flex"
-          ref={ref}
-          {...props}
-        >
+        <Flex alignItems="center" display="inline-flex" ref={ref} {...props}>
           <Box animation="spin" asChild size={size}>
             <svg
+              aria-label="Loading"
```

Comment:
let's move the label back on the flex - and add `role="status"` on the root flex element